spec/system/hiring_staff_can_sign_in_with_dfe_spec.rb

```diff
@@ -126,6 +126,41 @@
     end
   end
 
+  context "with DSI data including a school group (trust or local authority) that the school belongs to" do
+    let(:dsi_data) { { "trust_uids" => %w[14323], "school_urns" => %w[246757 953341 122792], "la_codes" => %w[323] } }
+    let!(:user) { create(:user, email: dsi_email_address, dsi_data: dsi_data) }
+    let(:school) { create(:school, urn: "246757") }
+
+    before do
+      allow(LocalAuthorityAccessFeature).to receive(:enabled?).and_return(true)
+
+      SchoolGroupMembership.create(school_group: school_group, school: school)
+
+      stub_authentication_step(school_urn: "246757", email: dsi_email_address)
+      stub_authorisation_step
+      stub_sign_in_with_multiple_organisations
+
+      visit root_path
+      sign_in_user
+    end
+
+    context "with trust" do
+      let(:school_group) { create(:trust, uid: "14323") }
+
+      scenario "it associates the user with the trust instead of the school" do
+        expect(current_path).to eql(organisation_managed_organisations_path)
```

Comment:
I quite like `expect(page).to have_current_path(...)`

Reply:
Also, is it worth checking the heading is for the Trust/LA respectively? (I think possibly).

And I also thought about whether we should test LA feature enabled/disabled? (I thought probably not because the flag will get removed soon)

Reply:
first one is a bit more concise though:
`expect(current_path).to eql(organisation_managed_organisations_path)`
vs
`expect(page).to have_current_path(organisation_managed_organisations_path)`

Reply:
> Also, is it worth checking the heading is for the Trust/LA respectively? (I think possibly).
added